src/Input/TextField/TextField.tsx

```diff
@@ -39,6 +46,12 @@ const TextField: React.FunctionComponent<Props> = props => {
     setInputType(inputType === 'password' ? 'text' : 'password');
   }, [setInputType, inputType]);
 
+  const handleClearInput = (e: React.MouseEvent<SVGSVGElement>) => {
+    const event = Object.create(e);
```

Comment:
Is this for cloning the event object? If so, can we use lodash's `cloneDeep`? That's more explicit about what's happening. Also I'm a bit scared that the docs on `Object.create` say that the input will be used as the "prototype" of the new object. I generally avoid the prototype system, so I don't know exactly if this is a problem or not.

Reply:
I think `Object.create` helps us to return the event object which includes `__proto__` same as native behavior. 
And actually I found this approach from `ant-design` sourcecode.